cfy_manager/components/__init__.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```

Comment:
plz move future imports to under the license header

it's funny, because in python there's quite a few things that are supposed to be "first thing in the file" (future imports, license, `# coding:`, maybe a shebang as well), but i think all actual code should be after the license

no hard reason for it, so apologies for commenting on essentially a style issue but yeah